fe2-android/app/src/main/java/com/github/dedis/popstellar/repository/LAORepository.java

```diff
@@ -25,11 +22,10 @@ public class LAORepository {
 
   private final HashMap<String, Lao> laoById = new HashMap<>();
   private final HashMap<String, Subject<Lao>> subjectById = new HashMap<>();
-  private final BehaviorSubject<Set<String>> laosSubject = BehaviorSubject.create();
+  private final BehaviorSubject<List<String>> laosSubject = BehaviorSubject.create();
```

Comment:
I put a list because this is always used in adapters where the order loosely matters. I agree that sorting by creation would be probably better.
How do you see it ? Since it is a collection of ids no elegant way to handle that comes to mind

Reply:
Alright thanks